test/TestSortFuser.cxx

```diff
@@ -18,16 +18,19 @@ using namespace care;
 using int_ptr = host_device_ptr<int>;
 
 CUDA_TEST(TestPacker, testFuseSort) {
-   care::initialize_pool("PINNED","PINNED_pool",chai::PINNED,128*1024*1024,128*1024*1024,true);
-   care::initialize_pool("DEVICE","DEVICE_pool",chai::GPU,128*1024*1024,128*1024*1024,true);
+#if defined(__CUDACC__)
```

Comment:
does this actually fail if __CUDACC__ is disabled? Do we want to enable it with CHAI_GPU_SIM_MODE ?